### PR TITLE
Fix bug: Incorrect arrows in DG

### PR DESCRIPTION
**Pull request by ekweirui**
Fix sequence diagram for `delete-person` in DG.

### CODE REVIEW

File: docs/diagrams/DeletePersonSequenceDiagram.puml [hunk 1]
```diff
@@ -100,8 +99,6 @@ activate CommandResult
 
 CommandResult --> DeletePersonCommand
 deactivate CommandResult
-CommandResult -[hidden]-> DeletePersonCommand
-destroy CommandResult
```
Comment (cheeheng) [inline docs/diagrams/DeletePersonSequenceDiagram.puml:102]: Add hidden arrow here? The intention is to make sure that the cross does not overlap with the activation bar.

**Review by cheeheng — COMMENTED** [review]
LGTM except for one nitpick.

File: docs/diagrams/DeletePersonSequenceDiagram.puml (hunk 1 above)
Comment (ekweirui) [reply docs/diagrams/DeletePersonSequenceDiagram.puml:102]: Great suggestion! I have since adopted it 💯

**Review by teoyuqi — APPROVED** [review]
LGTM

**Review by cheeheng — APPROVED** [review]
LGTM

**Pull request merged**